front/app/containers/UsersEditPage/messages.ts

```diff
@@ -119,6 +119,11 @@ export default defineMessages({
     id: 'app.containers.UsersEditPage.reasonsToStayListTitle',
     defaultMessage: 'Before you go...',
   },
+  activeProposalVotesWillBeDeleted: {
+    id: 'app.containers.UsersEditPage.activeProposalVotesWillBeDeleted',
+    defaultMessage:
+      'Your votes on proposals that are still open for voting will be deleted. Votes on proposals where the voting period has closed will not be deleted.',
```

Comment:
We briefly discussed it on a call with Koen, that maybe this modal should have more information about what will/won't be deleted. Maybe better to discuss with Simon F.

Also, maybe we should not (or should?) show this message if proposals are disabled?

Reply:
Probably, you remember, but just in case. We should not forget to make a custom copy specifically for Copenhagen, similar to this one https://github.com/CitizenLabDotCo/citizenlab/blob/acc23d5732707a7923a2df8b50f2bfd8a1221789/front/app/translations/da-DK.json#L1073

Reply:
Have we agreed custom copy for Copenhagen? Do you know where I can find it?

Reply:
Ah, yes. Got it. Yes, I still need to add in interpolation of `københavnerforslag`. Good point.

Reply:
I'm guessing we (or Søren) make the changes required for such Danish custom copy directly on Crowdin. Correct?

Reply:
>We briefly discussed it on a call with Koen, that maybe this modal should have more information about what will/won't be deleted.

I think we discussed that it could, and that this information could mention each proposal by name (or even with a link) where a user's vote will be deleted. It feels to me like that is a 'nice-to-have' that could come in a future iteration, however, and should not slow down getting this 'out-the-door'.

Reply:
> I'm guessing we (in conjunction with Søren) make the changes required for such Danish custom copy directly on Crowdin. Correct?

Yes, correct.